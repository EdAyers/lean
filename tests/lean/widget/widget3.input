{"seq_num": 0, "command": "sync", "file_name": "widget3.lean"}
{"seq_num": 46, "command": "info", "file_name": "widget3.lean", "line": 26, "column": 12}
{"seq_num": 47, "command" : "widget_event", "handler" : {"h":0, "r" : [15]}, "file_name": "widget3.lean", "line": 26, "column": 12, "args" : { "type" : "unit", "value" : {}}}
