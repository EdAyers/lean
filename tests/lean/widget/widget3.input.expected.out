{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"doc":"Repeatedly apply `intro1` and return the list of new local constants in order of introduction.","full-id":"tactic.intros","source":,"tactic_params":[],"type":"tactic (list expr)","widget":{"html":["0",{"attributes":null,"children":["+"],"events":{"onClick":{"handler":0,"route":[3]}},"tag":"button"},[{"attributes":null,"children":["0[ *** ][0]"],"tag":"div"},{"attributes":null,"children":["asdf"],"events":{"onClick":{"handler":1,"route":[12,3]}},"tag":"button"}]]}},"response":"ok","seq_num":46}
{"record":{"debug":{"column":2,"line":26,"msg":"from m_goal_pos"},"status":"success","widget":{"html":["1",{"attributes":null,"children":["+"],"events":{"onClick":{"handler":2,"route":[3]}},"tag":"button"},[{"attributes":null,"children":["1[ *** ][1, 0]"],"tag":"div"},{"attributes":null,"children":["asdf"],"events":{"onClick":{"handler":3,"route":[13,3]}},"tag":"button"}]]}},"response":"ok","seq_num":47}
